pom.xml

```diff
@@ -198,6 +198,16 @@
                 <artifactId>http-client</artifactId>
                 <version>${version.http-client}</version>
             </dependency>
+            <dependency>
+                <groupId>com.networknt</groupId>
+                <artifactId>client</artifactId>
```

Comment:
Do we need light-4j client module? If we include this one, we have the dependency for the Undertow.